readthedocs/builds/automation_actions.py

```diff
@@ -0,0 +1,2 @@
+def activate_version_from_regex_action(version, match_result, arg):
```

Comment:
Is it needed to be specific here on `from_regex_action`?

I suppose that this function should be just like `version.active = True` and `version.save()`. No need to use `match_result` nor `arg` here.

Reply:
All function actions will receive all the parameters. Match result and arg was explained above ^